### PR TITLE
Add SELinux variant to {busybox,f2fs-tools}

### PR DESCRIPTION
**Pull request by aparcar**
This patch accompanies the main [SELinux support](https://github.com/openwrt/openwrt/pull/3207) pull request by adding SELinux variants to both busybox and procd.

The move of Busyboxs config might be be a bit controversial.

### CODE REVIEW

File: package/utils/busybox/selinux.config [hunk 1]
```diff
@@ -0,0 +1,16 @@
+CONFIG_SELINUX=y
+CONFIG_FEATURE_TAR_SELINUX=y
+CONFIG_FEATURE_TAR_SELINUX=y
```
Comment (dangowrt) [inline package/utils/busybox/selinux.config:3]: No need to have that line twice I guess

File: package/utils/busybox/Makefile [hunk 2]
```diff
@@ -9,15 +9,15 @@ include $(TOPDIR)/rules.mk
...
+PKG_RELEASE:=3
 PKG_FLAGS:=essential
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=https://www.busybox.net/downloads \
 		http://sources.buildroot.net
 PKG_HASH:=d0f940a72f648943c1f2211e0e3117387c31d765137d92bd8284a3fb9752a998
 
-PKG_BUILD_DEPENDS:=BUSYBOX_CONFIG_PAM:libpam
+PKG_BUILD_DEPENDS:=BUSYBOX_CONFIG_PAM:libpam BUSYBOX_CONFIG_SELINUX:libselinux
```
Comment (dangowrt) [inline package/utils/busybox/Makefile:20]: I don't think we need to change that line when using variants. It's most likely a left-over from the previous approach just adding the config symbols.

File: package/utils/busybox/Makefile [hunk 3]
```diff
@@ -39,14 +39,27 @@ BUSYBOX_IF_ENABLED=$(if $(CONFIG_BUSYBOX_$(BUSYBOX_SYM)_$(1)),$(2))
...
-define Package/busybox
+define Package/busybox/Default
   SECTION:=base
   CATEGORY:=Base system
   MAINTAINER:=Felix Fietkau <nbd@nbd.name>
   TITLE:=Core utilities for embedded Linux
   URL:=http://busybox.net/
   DEPENDS:=+BUSYBOX_CONFIG_PAM:libpam +BUSYBOX_CONFIG_NTPD:jsonfilter
-  MENU:=1
+  PROVIDES:=busybox
```
Comment (dangowrt) [inline package/utils/busybox/Makefile:49]: Move the `PROVIDES:=busybox` to busybox-selinux please, as busybox itself doesn't need that.

Comment (aparcar) [reply package/utils/busybox/Makefile]: Done

**Comment by dangowrt** [comment]
Applied, thanks!

**Pull request closed without merging**